plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -681,6 +683,19 @@ public static function current_screen( $screen ) {
 		}
 	}
 
+	private static function maybe_redirect_to_new_marketplace_installer() {
+		// Redirect requires the "install" URL parameter to be passed.
+		if ( empty( $_GET[ 'install' ] ) ) {
+			return;
+		}
+
+		$my_subscriptions_url_base = self::get_helper_redirect_url( [], true );
```

Comment:
@bgrgicak If I remember you mentioned that it wasn't worth it since we will be removing the feature flag soon anyway. That said, I'm more than happy to implement this if you like.

Reply:
Seems like we are not removing it in 8.5, so it would be good to have the check.